tsconfig.json

```diff
@@ -1,24 +1,19 @@
 {
-  "include": ["src/global.d.ts", "src/**/*.ts", "__tests__/**/*.ts"],
+  "include": ["src/**/*.ts", "__tests__/**/*.ts"],
   "compilerOptions": {
-    "baseUrl": ".",
-    "rootDir": ".",
     "outDir": "dist",
     "sourceMap": false,
-    "noEmit": true,
 
     "target": "esnext",
     "module": "esnext",
     "moduleResolution": "node",
     "allowJs": false,
 
-    "noUnusedLocals": true,
```

Comment:
why disable this?

Reply:
because this is redundant and already covered by eslint. this will cause the same warning to be thrown twice